SEKOIA.IO Connector

yosradirgham opened the pull request:
In this PR we integrate SEKOIA.IO into Fortisoar SDK.
Please don't hesitate to reach out for any question!

Commit:
Add Fortisoar SDK

Commit:
Add health check + Add comment to alert

Commit:
integration: get an alert

Commit:
Get events Done

Commit:
update alert status

Commit:
import connector + list alerts

Commit:
Add actions for assets and actions for countermeasures + add test for get_alert

Commit:
Add more tests for alerts

Commit:
Add tests for assests

Commit:
Add tests

Commit:
final touch

cs-prerna-joshi reviewed (changes requested):
It seems that you have submitted integration SDK along with connector. 
Please only submit the sekoiaio folder.
You can take the reference of other connectors available on git.

squioc committed:
refactor: move the connector to the top-level

squioc committed:
fix: change import

squioc committed:
fix: add logo and update info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update get_alert.py

Fix get_alert file errors

yosradirgham committed:
Update sekoiaio/connector.py

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update sekoiaio/info.json

yosradirgham committed:
Update update_asset.py

Fix action update_asset

yosradirgham committed:
Update info.json

Fix info.json for action update_asset

yosradirgham committed:
Update activate_countermeasure.py

Fix url! Unfortunately the urljoin doesn't work as expected, that's why I was obliged to remove all the urljoin I was using! Maybe it's related to the python version we use???

yosradirgham committed:
Update sekoiaio/activate_countermeasure.py

yosradirgham committed:
Update sekoiaio/deny_countermeasure.py

yosradirgham committed:
Update sekoiaio/deny_countermeasure.py

yosradirgham committed:
Merge pull request #1 from SEKOIA-IO/fix/Connector

Apply Fortinet recommandations
